flags.html

```diff
@@ -5,79 +5,79 @@
     </head>
     <body>
 <div id="indonesia" class="flagContainer">
-
+    <div class="indonesiaRed"></div>
```

Comment:
this could be dangerous.  basically you are relying on your background to be white.  That's a bad idea.  Please fix this.  If the background turned a different color, your element's background isn't white, it's transparent.